### DISCUSSION TITLE
Error "The string to escape is not a valid UTF-8 string" when open Performance toolbar

### DISCUSSION THREAD

**Discussion by ad3n** [symfony/symfony#45086] [Q&A] [answered]
### Symfony version(s) affected

5.4/6.0

### Description

Error "The string to escape is not a valid UTF-8 string" when open performance toolbar

<img width="1066" alt="Screen Shot 2022-01-20 at 09 46 39" src="https://user-images.githubusercontent.com/7464920/150263267-0ecf7dac-d677-4617-bc6b-af01e38c2baa.png">

### How to reproduce

Show profiler -> select Performance

**Comment by derrabus** [comment, marked as answer]
Moving to support. Without a reproducer, it's not a bug.

**Reply by ad3n** [reply]
Hi @derrabus,

I found, the error only occur when i visit from specific page, i think that is from my data, i will check again

Thank you for your reply
